assets/css/style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+    --paddi: 10%;
```

Comment:
Is this a typo? If it's a naming thing, it looks like a typo, as it's an incomplete word. If you are trying to avoid using `padding` to avoid confusion, you can use something like `spacing`, or even better, something to include spacing + the value - in this case, 10% (in bigger projects, you might have multiple variables declared like this, one for 10%, one for 25%, etc..)